### PR TITLE
VB-729: Booking journey Cypress tests

### PR DESCRIPTION
**Pull request by tpmcgowan**
* Refactor of integration test Wiremock stubs to reflect API changes
* Use common test objects from `TestData.ts`
* Add test for Home page tiles
* Add full booking journey test
* Update existing prisoner search tests
* Split off Prisoner profile page tests

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -119,7 +119,7 @@ jobs:
           command: sudo apt-get install libxss1
       - run:
           name: Get wiremock
-          command: curl -o wiremock.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-standalone/2.27.1/wiremock-standalone-2.27.1.jar
+          command: curl -o wiremock.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-jre8-standalone/2.35.0/wiremock-jre8-standalone-2.35.0.jar
```
Comment (tpmcgowan) [inline .circleci/config.yml:122]: Upgrading to latest Wiremock version so it matches what runs locally in Docker from the `wiremock/wiremock` image. Older version here was causing builds in CircleCI to fail.

Comment (psoleckimoj) [reply .circleci/config.yml:122]: Is it possible to fix the wiremock image to a version? So we can upgrade when we choose?

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: .circleci/config.yml (hunk 1 above)
Comment (tpmcgowan) [reply .circleci/config.yml:122]: Good suggestion! I've set it to the same `2.35.0` tag in `docker-compose-test.yml` and commented in both files as a reminder that the versions should match.

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**